app/controllers/application_controller.rb

```diff
@@ -1,4 +1,12 @@
 class ApplicationController < ActionController::Base
   layout 'application'
   include Backable
+
+  rescue_from ActiveRecord::RecordNotFound, with: :page_not_found
+
+  private
+
+  def page_not_found
+    redirect_to error_path(:page_not_found)
+  end
```

Comment:
As "page not found" is a generic error, I've hooked that in at the lowest level - so this is the default error anywhere in the app for `ActiveRecord::RecordNotFound`.